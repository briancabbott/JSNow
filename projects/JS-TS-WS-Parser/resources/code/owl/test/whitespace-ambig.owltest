---
.whitespace 'x'
x = y y
y = 'a' | 'a' 'a'
