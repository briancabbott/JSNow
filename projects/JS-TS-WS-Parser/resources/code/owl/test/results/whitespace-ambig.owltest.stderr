error: this grammar is ambiguous

. axaxax

  can be parsed in two different ways: as

. axaxax
  y-- y 
  x---- 

  or as

. axaxax
  y y-- 
  x---- 

